scriptworker/cot/verify.py

```diff
@@ -1395,7 +1395,7 @@ def compare_jsone_task_definition(parent_link, compare_definition):
             mach_commands = ACTION_MACH_COMMANDS
         else:
             mach_commands = DECISION_MACH_COMMANDS
-        verify_decision_command(link, mach_commands=mach_commands)
+        # verify_decision_command(link, mach_commands=mach_commands)
```

Comment:
This check should go away when we kill cotv1.

Reply:
Good idea. I'll do it this way.